src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+
+    @Override
+    public void updateDataBase(FruitTransaction transaction) {
+        if (!transaction.getFruit().equals(BANANA) && !transaction.getFruit().equals(APPLE)) {
```

Comment:
The current implementation restricts the operation to only 'banana' and 'apple'. Consider using a more flexible approach, such as checking against a configurable list of valid fruits, to allow for easier updates and maintenance.